book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(date, self, person)
+    @rentals.push(rental)
```

Comment:
- [ ] You are pushing rental into the rentals array but the rental does not exist. Please create a variable rental. It can look something like this.
```
    rental = Rental.new(date, self, person)
    @rentals.push(rental)
```